tests/call/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -128,14 +142,17 @@ $db->Execute($sql, [
     $somewhatLongParameterXYZ
 ]);
 
-$db->Execute([
-    $foo,
-    $bar,
-    $foobar,
-    $somewhatLongParameter,
-    $somewhatLongParameterX,
-    $somewhatLongParameterXYZ
-], $sql);
+$db->Execute(
+    [
+        $foo,
+        $bar,
+        $foobar,
+        $somewhatLongParameter,
+        $somewhatLongParameterX,
+        $somewhatLongParameterXYZ
+    ],
+    $sql
+);
```

Comment:
Looks PSR-12 say what we can don't write each argument on newline. Or i am wrong?

Reply:
im not sure, I couldn't find anything... do you know where it says that? This is how prettierjs outputs it

Reply:
> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.

> When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.

https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#45-method-and-function-arguments

Seems fine to me.

Reply:
maybe @evilebottnawi was thinking of this sentence: 

> A single argument being split across multiple lines (as might be the case with an anonymous function or array) does not constitute splitting the argument list itself.

in:

https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#47-method-and-function-calls

Reply:
You're right @nicoder - I think this is at least part of the reason our first implementation was a bit different than the one in JS.

Reply:
@mgrip What'

Reply:
i interpret that as you _can_ split an argument across multiple lines while keeping the others on a single line, but also

> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line

From reading the psr12 spec im not seeing that there's any rhyme or reason to when you split vs keep the arguments on the same line (which is good because imo we should follow the prettierjs logic)

Reply:
ie by psr12 standards i think both

```php
$db->Execute([
    $foo,
    $bar,
    $foobar,
    $somewhatLongParameter,
    $somewhatLongParameterX,
    $somewhatLongParameterXYZ
], $sql, $someOtherArg);
```

_and_

```php
$db->Execute(
  [
      $foo,
      $bar,
      $foobar,
      $somewhatLongParameter,
      $somewhatLongParameterX,
      $somewhatLongParameterXYZ
  ],
  $sql,
  $someOtherArg
);
```
 would be acceptable

Reply:
@mgrip can we open issue about this for discussion?

Reply:
Some of the examples mentioned in the style guides seem to prefer the "shorter" version, but I agree that the "longer" style doesn't seem to be explicitly forbidden. Probably it's the easiest to stay close to the JS implementation and move forward like this?